README.md

```diff
@@ -0,0 +1,23 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Hello Microverse
+
+> This is my first Microverse project.
+
+
+## Built With
+
+- Major languages: html, css
+
```

Comment:
- [ ] Kindly add the getting started section, this will make your readme more professional and help run the project locally.

Reply:
- [ ] [OPTIONAL] It would be nice to use [this template ](https://github.com/microverseinc/readme-template) to add more sections to your readme.